Update Past Events

Jeymz opened the pull request:
Updated the past events page to include:
- June 2022 Meeting
- November 2022 Meeting + Slides

Commit:
Past Events Update

Updated the past events page to include
 - June 2022 Meeting
 - November 2022 Meeting + Slides

Commit:
Merge remote-tracking branch 'origin/past_events_update'

szh commented on tab_pastevents.md:
```diff
@@ -12,13 +12,35 @@ tags: cincinnati
+**Speaker Biography:** Hi! My name is Shlomo Heigh and I'm a senior software engineer at CyberArk. I maintain the Conjur open source secrets management application, primarily working on integrations with platforms like Kubernetes and engaging with the open source community.  Before that, I was a full stack developer for a SaaS firm. I've worked with everything from Ruby to Go and am always lookin ...
+
+**Presentation:**
+TBD
```
Here you go
[Conjur Secrets Management.pdf](https://github.com/OWASP/www-chapter-cincinnati/files/10723062/Conjur.Secrets.Management.pdf)

Jeymz replied:
Updated pull request to include this document.

szh commented on tab_pastevents.md:
```diff
@@ -12,13 +12,35 @@ tags: cincinnati
+**Speaker Biography:** Jay Simmons is a senior application security analyst at Great American Insurance Group who is responsible for their security champions program. 
+
+**Presentation:*
+[OWASP Cinci - November 2022 - Security Champions Program](https://owasp.org/www-chapter-cincinnati/assets/presentations/Maintaining%20a%20Security%20Champions%20Program.pdf)
```
Will this link automatically work once this PR is merged?

Jeymz replied:
I think it would, I tested this with the current assets to verify that url path matches, but I can use relational links if that would make more sense.

Jeymz replied:
Updated pull request to include relative links instead of strict links and also updated gemfile and gitignore to support local development rendering.

Commit:
Local Rendering and Link Path Updates

- Updated gemfile and gitignore to support local development environment rendering
- Added PDF of presentation from June 2022
- Updated link paths to be relative instead of strict

Commit:
Merge remote-tracking branch 'origin/past_events_update'

szh reviewed (approved):
LGTM! Will need @leisringowasp to merge

Commit:
Formatting Correction

Commit:
Merge remote-tracking branch 'origin/past_events_update'
